html/includes/forms/alert-transports.inc.php

```diff
@@ -62,6 +62,7 @@
     $status = 'error';
     $message = 'Missing transport information';
 } else {
+    $is_default = $is_default ? 1 : 0;
```

Comment:
I think it would be better if you did this up above (line 53 & 55).

Reply:
That's a good suggestion, I'll make that change.